src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -4,128 +4,126 @@
+register_cli_argument(
+    'sf cluster node', 'number_of_nodes_to_remove', help='number of nodes to remove.')
 
 register_cli_argument('sf cluster durability', 'durability_level',
-                      help='durability level.',**enum_choice_list(['Bronze', 'Silver', 'Gold']))
+                      help='durability level.', **enum_choice_list(['Bronze', 'Silver', 'Gold']))
 
-register_cli_argument('sf cluster setting', 'parameter',
-                      help='parameter name')
-register_cli_argument('sf cluster setting', 'section',
-                      help='section name')
-register_cli_argument('sf cluster setting', 'value',
-                      help='Specify the value')
+register_cli_argument('sf cluster setting', 'parameter', help='parameter name')
+register_cli_argument('sf cluster setting', 'section', help='section name')
+register_cli_argument('sf cluster setting', 'value', help='Specify the value')
 
-register_cli_argument('sf cluster setting', 'settings_section_description',
-                      help='Specify the value')
+register_cli_argument('sf cluster setting',
+                      'settings_section_description', help='Specify the value')
 
-register_cli_argument('sf cluster upgrade-type set', 'version',
-                      help='cluster code version')
+register_cli_argument('sf cluster upgrade-type set',
+                      'version', help='cluster code version')
 register_cli_argument('sf cluster upgrade-type set', 'upgrade_mode',
-                      help='cluster upgrade mode',**enum_choice_list(['manual', 'automatic']))
+                      help='cluster upgrade mode', **enum_choice_list(['manual', 'automatic']))
 
 register_cli_argument('sf cluster reliability', 'reliability_level',
-                      help='durability level.',**enum_choice_list(['Bronze', 'Silver', 'Gold']))
+                      help='durability level.', **enum_choice_list(['Bronze', 'Silver', 'Gold']))
 
 register_cli_argument('sf cluster reliability', 'auto_add_node',
                       help='Add node count automatically when changing reliability.')
 
-register_cli_argument('sf cluster setting set','settings_section_description', type=get_json_object,
-               help='JSON encoded parameters configuration. Use @{file} to load from a file.'
-               """for example: [{"section": "NamingService","parameter": "MaxOperationTimeout","value": 1000},{"section": "MaxFileOperationTimeout","parameter": "Max2","value": 1000]""")
+register_cli_argument('sf cluster setting set', 'settings_section_description', type=get_json_object,
+                      help='JSON encoded parameters configuration. Use @{file} to load from a file.'
+                      """for example: [{"section": "NamingService","parameter": "MaxOperationTimeout","value": 1000},{"section": "MaxFileOperationTimeout","parameter": "Max2","value": 1000]""")  # pylint: disable=line-too-long
```

Comment:
This would be more appropriate for the parameter long-summary in the helps.py file.